src/main/java/no/nav/finnkandidatapi/midlertidigutilgjengelig/MidlertidigUtilgjengeligController.java

```diff
@@ -58,6 +60,15 @@ private boolean datoErMerEnn30DagerFremITid(LocalDateTime tilDato) {
         return ok(dto);
     }
 
+    @PostMapping("/aktører")
```

Comment:
POST for å hindre aktørid i urlen?

Reply:
Post fordi jeg har spørring. 
Get er mest semantisk riktig for spørringer, men get har følgende potensielle problemer:
- Get med url vil feile om aktørid listen er lang, det er et maks antall tegn i url, avhengig av server/webrammeverk
- Potensielt manglende støtte for body i web rammeverk(spring) i finn-kandidat-api ved get.
- Potensielt manglende støtte for body i klient ved get.
- Eksponering av sensitiv id i url.
- Url encoding kan bli glemt.